concert/cloudformation.yml

```diff
@@ -383,6 +386,117 @@ Resources:
         - DynamoDBCrudPolicy:
             TableName: !Ref Table
 
+  AggregateChoiceDataStream:
+    Type: AWS::KinesisFirehose::DeliveryStream
+    Properties:
+      DeliveryStreamName: !Sub ${Environment}-AggregateChoiceDataStream
+      S3DestinationConfiguration:
+        BucketARN: !GetAtt AggregateChoiceDataS3Bucket.Arn
+        BufferingHints:
+          IntervalInSeconds: 60
+          SizeInMBs: 50
+        CompressionFormat: UNCOMPRESSED
+        RoleARN: !GetAtt AggregateChoiceDataStreamRole.Arn
+        CloudWatchLoggingOptions:
+          Enabled: true
+          LogGroupName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+          LogStreamName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+
+  AggregateChoiceDataStreamLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+      RetentionInDays: 30
+  AggregateChoiceDataStreamLogStream:
+    Type: AWS::Logs::LogStream
+    Properties:
+      LogGroupName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+      LogStreamName: s3Backup
+
+  AggregateChoiceDataStreamRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: firehose.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: firehose_delivery_policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: s3:*
+                Resource: '*'
```

Comment:
:open_mouth: 

No god mode, please

Reply:
😂

Yeah, I meant to add an explanatory comment here. When using `!GetAtt AggregateChoiceDataS3Bucket.Arn`, I couldn't get this policy to properly grant access to the bucket. I'm still not totally sure why that was happening, but it seemed like the workaround was probably to specify the Arn using the `!Sub` pattern used elsewhere above. I thought that that meant I'd probably need to define a bucket name for the S3 resource, and since bucket names are globally unique (right?), I didn't want to add that update to my template and steal our bucket names without getting your take first